src/exe/FreeC/Application/Options.hs

```diff
@@ -65,4 +69,6 @@ makeDefaultOptions = do
                    , optCreateCoqProject          = True
                    , optTransformPatternMatching  = False
                    , optDumpTransformedModulesDir = Nothing
+                   , optFrontend                  = "haskell"
+                   , optBackend                   = "coq"
```

Comment:
The names of the default front end and back end should probably not be hard coded. Try to use the `name` field of `haskellFrontend` and `coqBackend`.